workspaces/arborist/test/fixtures/reify-cases/conflict-bundle-file-dep.js

```diff
@@ -0,0 +1,23 @@
+// generated from test/fixtures/conflict-bundle-file-dep
+module.exports = t => {
+  const path = t.testdir({
+    "fixtures-has-package-exports": {
+      "package.json": JSON.stringify({
+        "bundleDependencies": true,
+        "dependencies": {
+          "abbrev": "1.1.1"
```

Comment:
Another interesting part of this is that having any dep here is load bearing to the test failing. I made it abbrev since it already exists as a fixture and has no other deps, but any dep here fails, but removing `dependencies` altogether passes.